vip-helpers/vip-utils.php

```diff
@@ -1037,7 +1037,7 @@ function wpcom_vip_load_plugin( $plugin = false, $folder = false, $load_release_
 	}
 
 	if ( $includepath && file_exists( $includepath ) ) {
-		wpcom_vip_add_loaded_plugin( "{$plugin_type}/{$plugin}" );
+		wpcom_vip_add_loaded_plugin( "{$plugin_type}/{$plugin}/{$file}" );
```

Comment:
Why is this change needed?

Reply:
The data we get back from `wpcom_vip_get_loaded_plugins()` is much closer to the option we store in `active_plugins` with this change. It makes comparing the two a little more accurate and easier.

I compare them here: https://github.com/Automattic/vip-dashboard/pull/75/files#diff-89ad60d19282ec7bc4917b8a060268ccR392